src/main/java/javax/jmdns/impl/JmDNSImpl.java

```diff
@@ -68,6 +68,9 @@
  * @author Arthur van Hoff, Rick Blair, Jeff Sonstein, Werner Randelshofer, Pierre Frisch, Scott Lewis, Kai Kreuzer, Victor Toni
  */
 public class JmDNSImpl extends JmDNS implements DNSStatefulObject, DNSTaskStarter {
+
+    private static final boolean IS_WINDOWS = System.getProperty("os.name").startsWith("Windows");
```

Comment:
Better check `System.getProperty("os.name")` for `null` to be on the safe side.

Reply:
Thank you! Fixed